### PR TITLE
Add page documenting Gitea as a Windows service

### PR DESCRIPTION
**Pull request by andreynering**

### CODE REVIEW

File: content/doc/installation/windows-service.en-us.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    identifier: "windows-service"
+---
+
+# Register as a Windows Service
+
+To register Gitea as a Windows service, first run `cmd` as an Administrator,
+and then run the following command:
+
+```
+sc create gitea start= auto binPath= ""C:\gitea\gitea.exe" web --config "C:\gitea\custom\conf\app.ini""
```
Comment (tboerger) [inline content/doc/installation/windows-service.en-us.md:21]: Too much spaces and quotes?

**Comment by tboerger** [comment]
Please drop the linebreaks within the paragraphs and add a slug attribute to the top

File: content/doc/installation/windows-service.en-us.md (hunk 1 above)
Comment (andreynering) [reply content/doc/installation/windows-service.en-us.md:21]: It's by design. Won't work, otherwise.

**Comment by andreynering** [comment]
> Please drop the linebreaks within the paragraphs

So a single line per paragraph?

**Comment by tboerger** [comment]
Yes, that's what we have for all the docs, blog and so on

**Comment by andreynering** [comment]
I definitely prefer the convention of wrapping by 80 characters. But no problem, I updated. :+1:

**Comment by tboerger** [comment]
LGTM

**Comment by thibaultmeyer** [comment]
LGTM

**Pull request merged**